parm/config/config.fv3

```diff
@@ -86,9 +92,11 @@ case $case_in in
         export layout_y=8
         export layout_x_gfs=6
         export layout_y_gfs=6
-        export nth_fv3=1
-        export npe_node_fcst=$(echo "$npe_node_max/$nth_fv3" |bc)
+        export npe_wav=270
+        export nth_fv3=4
+        export nth_efcs=1
```

Comment:
remove this line

Reply:
Remove which line?

Can I remove all npe_node_fcst calculation lines from config.fv3? They were added in feature/gfsv16b but I prefer those lines being in config.resources for the final calculation of npe_node_fcst.